packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -405,31 +402,89 @@ class Treemap extends StatelessWidget {
           ],
         ),
       );
+    } else {
+      return Column(
+        children: [
+          if (levelsVisible == 2 && height > minHeightToDisplayTitleText)
```

Comment:
Why are we special casing 2 here? see other comment about adding an outerLevel param. Don't we want a title even for level 1 nodes?

Reply:
Restructured code so this line is deleted.